cypress/integration/parallel-5/discussion.spec.js

```diff
@@ -17,12 +17,20 @@ describe('Discussion', function () {
 
 	it('should scroll the page to the comments section and expand it when the comment count link is clicked', function () {
 		cy.visit(`/Article?url=${articleUrl}`);
+		cy.task(
```

Comment:
Without these logs the tests for this just hang on GH Actions.

Reply:
I wonder if this is related to the `only` above?

Reply:
It shouldn't be because it's in a totally different test. I've removed anyway and we'll see what happens